### PR TITLE
Feature/sc 381

### PR DESCRIPTION
**Pull request by DanCh11**

### CODE REVIEW

**Review by CaptainOfHacks — CHANGES_REQUESTED** [review]
The code on the current branch needs to be updated with the code on the main branch and to resolve conflicts between branches. Added tests for written code.

File: sem_covid/entrypoints/ml_dags/classifiers_dag.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```
Comment (CaptainOfHacks) [inline sem_covid/entrypoints/ml_dags/classifiers_dag.py:1]: This file is not related to SC-381.

File: sem_covid/entrypoints/notebooks/language_modeling/language_model_tools/similarity_calculus.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import pandas as pd
+from gensim.models import KeyedVectors
+from sklearn.metrics import pairwise_distances
+
+
+def euclidean_similarity(vector_1: np.array, vector_2: np.array) -> np.float:
+    """
+        calculate euclidean distance between two vectors
```
Comment (CaptainOfHacks) [inline sem_covid/entrypoints/notebooks/language_modeling/language_model_tools/similarity_calculus.py:9]: calculate euclidean similarity ( not distance )

File: sem_covid/entrypoints/notebooks/language_modeling/language_model_tools/similarity_calculus.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    """
+    return 1 / (1 + np.linalg.norm(vector_1 - vector_2))
+
+
+def cosine_similarity(vector_1: np.array, vector_2: np.array) -> np.float:
+    """
+        calculate cosine distance between two vectors
```
Comment (CaptainOfHacks) [inline sem_covid/entrypoints/notebooks/language_modeling/language_model_tools/similarity_calculus.py:16]: calculate cosine similarity

File: sem_covid/entrypoints/notebooks/language_modeling/language_model_tools/similarity_calculus.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    """
+    return np.dot(vector_1, vector_2) / (np.linalg.norm(vector_1) * np.linalg.norm(vector_2))
+
+
+def manhattan_similarity(vector_1: np.array, vector_2: np.array) -> np.float:
+    """
+        calculate manhattan distance between two vectors
```
Comment (CaptainOfHacks) [inline sem_covid/entrypoints/notebooks/language_modeling/language_model_tools/similarity_calculus.py:23]: calculate manhatan similarity

File: sem_covid/entrypoints/notebooks/language_modeling/language_model_tools/similarity_calculus.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    """
+    return 1 / (1 + np.sum(np.abs(vector_1 - vector_2)))
+
+
+# def get_similarity_matrix(wv: KeyedVectors, similarity_function) -> pd.DataFrame:
```
Comment (CaptainOfHacks) [inline sem_covid/entrypoints/notebooks/language_modeling/language_model_tools/similarity_calculus.py:28]: If we need the code in the comments, a function must be made for it. If we don't need this, just delete this comments.

**Review by CaptainOfHacks — CHANGES_REQUESTED** [review]
Some files are not update from main branch.
sem_covid/entrypoints/ml_dags/classifiers_dag.py -- this file is not related to SC-381.
Update documentation and clean code from comments.

File: sem_covid/entrypoints/ml_dags/classifiers_dag.py (hunk 1 above)
Comment (DanCh11) [reply sem_covid/entrypoints/ml_dags/classifiers_dag.py:1]: to update the project and resolve the conflicts, i pushed the file from the main in this branch. Those 100+ files are updated files from the main. And this one is part of them.

**Review by CaptainOfHacks — APPROVED** [review]
